FiveCalls/FiveCalls/Actions.swift

```diff
@@ -7,35 +7,13 @@
 //
 import Foundation
 
-protocol Action { }
-
-struct FetchIssuesAction: Action {}
-
-struct SetIssuesAction: Action {
-    let issues: [Issue]
-}
-
-struct FetchContactsAction: Action {
-    let location: NewUserLocation
-}
-
-struct SetContactsAction: Action {
-    let contacts: [Contact]
-}
-
-struct SetLocationAction: Action {
-    let location: NewUserLocation
+enum Action {
```

Comment:
This will work as long as we only have a single action type, which may be the case with an app this small. I think it makes sense at this point for simplicity 👍🏽